front/src/utils/date.ts

```diff
@@ -242,3 +242,22 @@ export const formatDateString = (date?: Date | null) => {
   if (!date) return '';
   return dayjs(date).format('DD/MM/YY');
 };
+
+const parseDate = (dateString: string): Date => {
```

Comment:
I think we can use dayjs here

Reply:
done, thank you for the comment, it saves many lines :)